tests/testthat/test-ui-chord.R

```diff
@@ -3,10 +3,14 @@ context("canvasXpress Web Charts - Chord")
 
 test_that("cXchord1", {
     check_ui_test(cXchord1())
+
+    warning("Hovering over legend fade complete plot")
 })
 
 test_that("cXchord2", {
     check_ui_test(cXchord2())
+
+    warning("Hovering over legend fade complete plot")
```

Comment:
please update the message to 'Hovering over the legend fades the entire plot' for clarity. Thanks